parlai/core/torch_agent.py

```diff
@@ -263,6 +263,16 @@ def add_cmdline_args(cls, argparser):
         agent.add_argument(
             '-tr', '--truncate', default=-1, type=int,
             help='Truncate input lengths to increase speed / use less memory.')
+        agent.add_argument(
+            '--left-truncate', type=int,
```

Comment:
nit: can we call them text-truncate and cand-truncate? when I hear "left-truncate", I think of which side I'm cutting off. You could imagine left-truncating the label so only its final words are showing.